onnxscript/converter.py

```diff
@@ -902,6 +936,7 @@ def translate_opset_expr(self, node: ast.Attribute) -> values.Opset:
         else:
             self.fail(node, "Invalid opset expression.")
 
+    # pylint: enable=inconsistent-return-statements
```

Comment:
Be sure to enable it afterwards? Or move to the file top to disable it for the whole file

Reply:
My bad :)